packages/pwa-kit-runtime/src/utils/logger.test.js

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import createLogger, {PWAKITLogger} from './logger'
+
+describe('PWAKITLogger', () => {
```

Comment:
FYI some tests are failing. But overall, great tests though.. thanks for writing them!